src/Razor/src/Microsoft.VisualStudio.Editor.Razor/ContextChangeKind.cs

```diff
@@ -9,5 +9,6 @@ public enum ContextChangeKind
         EditorSettingsChanged,
         TagHelpersChanged,
         ImportsChanged,
+        TextViewsChanged,
```

Comment:
This is super questionable but see: https://github.com/aspnet/AspNetCore-Tooling/pull/112/files#diff-c23a63f829806e9af705514131701d88R122 for why.

Reply:
I'm completely fine with this as hack, because this area is already a hack.. What is it that they are listening to and could we simplify it? It seems like they know us too well.